examples/fill_in_reduction.rs

```diff
@@ -38,7 +38,7 @@ fn small_lap_mat() -> sprs::CsMat<f64> {
 }
 
 fn lap_mat_from_obj(path: &str) -> Result<sprs::CsMat<f64>, tobj::LoadError> {
-    let (objects, _materials) = tobj::load_obj(&Path::new(path))?;
+    let (objects, _materials) = tobj::load_obj(&Path::new(path), false)?;
```

Comment:
The laplacian matrix is built from triangles, so I think the `triangulate` flag passed to `load_obj` should be set to `true`.